src/modules/storage.js

```diff
@@ -1,20 +1,24 @@
 const listsList = document.querySelector('.today-lists');
+// const moveDelete = document.querySelector('.single-list-action-button');
+// const listValue = document.querySelector('.single-list-input');
 
-let listArray = [];
+export let listArray = []; // eslint-disable-line
 
-const addList = () => {
+/* eslint-disable no-unused-vars */
+
+export const addList = () => {
   let listCode = '';
   listArray.forEach((element, index) => {
     const { list } = element;
     if (index % 2 === 0) {
       listCode += `
-        <div class="single-list div-style">
+        <div class="single-list div-style" id="${index}">
           <form class="single-list-form">
             <input type="checkbox" class="checkbox">
-            <input type="text" id="single-list-item" class="single-list-input main-inputs" value="${list}">
+            <input type="text" class="single-list-input main-inputs" value="${list}">
           </form>
           <div class="single-list-action-button">
-          <button class = "move-btn"><i class="fa-solid fa-ellipsis-vertical"></i></button>
+            <button class = "delete-btn" onclick="removeList('${list}')"><i class="fa-solid fa-trash-can"></i></button>
```

Comment:
- [ ] Kindly review the add function as the items are not getting added to either the localStorage or getting displayed on screen
- [ ] Please ensure that all changes are saved in the localStorage.